packages/flutter/lib/src/widgets/actions.dart

```diff
@@ -72,6 +72,29 @@ typedef ActionListenerCallback = void Function(Action<Intent> action);
 /// The [ActionDispatcher] can invoke an [Action] on the primary focus, or
 /// without regard for focus.
 ///
+/// ### Action Overriding
+///
+/// An overridable [Action] is a special kind of [Action] created using the
+/// [Action.overridable] constructor. It has access to a default [Action],
+/// and a nullable override [Action]. It the same behavior as its override
+/// if that exists, and mirrors the behavior of its `defaultAction` otherwise.
+///
+/// The [Action.overridable] constructor creates overridable [Action]s that use
+/// a [BuildContext] to find a suitable override in its ancestor [Actions]
+/// widget. This can be used to provide a default implementation when creating a
+/// general purpose leaf widget, and later override it when building a more
+/// specialized widget using that widget.
+///
+/// For instance, [TextField]'s [SelectAllTextIntent] by default selects the
```

Comment:
We really need to include this as a sample. In the sample you could also override the actions bound to backspace and forward backwards cursor, etc.

Reply:
There's one for the factory constructor that overrides `DeleteTextIntent`